resources/assets/uad_lists.json

```diff
@@ -12237,7 +12237,7 @@
     "dependencies": null,
     "neededBy": null,
     "labels": null,
-    "removal": "Expert"
+    "removal": "Recommended"
```

Comment:
The base package here is in Advanced. If you decide that themes shouldn't be categorized separately to what they theme, you should make them both the same(Recommended/Advanced, whichever you go for).
But as I've alluded to before, I do think themes should be separated from what they theme, especially since some OEMs add theme packages for core AOSP packages.

Reply:
You're right.